README.md

```diff
@@ -31,9 +31,17 @@ scoop bucket add svix https://github.com/svix/scoop-svix.git
 scoop install svix
 ```
 
-### Other
+### Linux
 
-If you are on another platform such as linux or just rather not use a package manager, you can download a binary from our Github releases and use it straight away without having to install any additional dependencies.
+The Svix CLI is available on Linux via the [snap](https://snapcraft.io/) package manager:
```

Comment:
And in AUR: https://aur.archlinux.org/packages/svix-cli/

Reply:
@tasn please confirm that that is the correct pacman command, its been a while since I've used arch 😅